src/sentry/projectoptions/defaults.py

```diff
@@ -184,3 +184,6 @@
     key="sentry:loader_available_sdk_versions",
     epoch_defaults={1: ["8.x", "7.x", "6.x", "5.x", "4.x"], 11: ["8.x", "7.x"]},
 )
+
+# Dynamic sampling rate in project-level "manual" configuration mode
+register(key="sentry:target_sample_rate", default=1.0)
```

Comment:
Technically, we wouldn't need a default as we could default this in the DS logic. However, to be more consistent and provide proper values to the UI, I'm adding this default here.